src/openrct2/world/footpath.c

```diff
@@ -346,8 +347,19 @@ static money32 footpath_place_real(sint32 type, sint32 x, sint32 y, sint32 z, si
         return MONEY32_UNDEFINED;
     }
 
-    if (flags & GAME_COMMAND_FLAG_APPLY)
-        footpath_interrupt_peeps(x, y, z * 8);
+    if ((flags & GAME_COMMAND_FLAG_APPLY)) {
```

Comment:
I still wonder if its part of the original game to behave like that or some actual bug that shouldn't exist, for offline gameplay its not a game breaking thing.